test/data/javascript/tests.js

```diff
@@ -2576,6 +2576,7 @@ exports.test_data = {
         { unchanged: 'if{{s}}(a) b()' },
         { unchanged: 'while{{s}}(a) b()' },
         { unchanged: 'do\n    c();\nwhile{{s}}(a) b()' },
+        { unchanged: 'switch{{s}}(a) b()' },
```

Comment:
I would prefer this to be a valid `switch`/`case` statement, but the formatting works so... okay.